src/list.tsx

```diff
@@ -269,6 +271,7 @@ interface ToRowContentProps extends CommonProps {
     to: string | undefined;
     fullPageOnWebView?: boolean;
     replace?: boolean;
+    onNavigate?: () => void | Promise<void>;
```

Comment:
Why not generalizing using the new Touchable type that you created?

Reply:
This case is more complex because there are more cases (controls, iconButton, etc), although it may be possible to use the type util here too. Also, I have two other PRs changing this same component, and one of them changing types too. I don't want conflicts.